README.md

```diff
@@ -20,43 +12,40 @@ The Titanium CLI requires [Node.js 0.10.x](http://nodejs.org/dist/) or newer.
 
     [sudo] npm install -g titanium
 
+After install, Titanium CLI is executable as `ti`    
+
 ## Bleeding Edge
 
-You can download the latest and greatest *unstable* Titanium CLI code by
-running the following:
+You can download the latest and greatest *unstable* Titanium CLI code by running the following:
 
-    [sudo] npm install -g git://github.com/appcelerator/titanium.git
+    [sudo] npm install -g git://github.com/tidev/titanium.git
 
 ## Obtaining a Titanium SDK
 
-You will need to download a Titanium SDK 3.0 or newer:
+You will need to download a Titanium SDK:
 
     # stable release (recommended)
-    titanium sdk install --default
+    ti sdk install --default
 
-There are times when the latest stable release has an issue or lacks a feature
-in which case you may want to try an unstable Titanium SDK build. These builds
-are not recommended for production use, but should work for development.
+There are times when the latest stable release has an issue or lacks a feature in which case you may want to try an unstable Titanium SDK build. These builds are not recommended for production use, but should work for development.
 
     # unstable next release
-    titanium sdk install --branch 3_1_X --default
+    ti sdk install --branch 3_1_X --default
 
     # bleeding edge
-    titanium sdk install --branch master --default
+    ti sdk install --branch master --default
 
 ## Setting up the Titanium CLI
 
-Before you begin using the Titanium CLI, you should configure it by running the
-"setup" command:
+Before you begin using the Titanium CLI, you should configure it by running the "setup" command:
 
-    titanium setup
+    ti setup
 
-It will prompt you with a number of questions such as your e-mail address (for
-logging into the Appcelerator Network), the location of your Android SDK, and so on.
+It will prompt you with a number of questions such as your e-mail address (for logging into the Appcelerator Network), the location of your Android SDK, and so on.
```

Comment:
You can remove the e-mail address option. We haven't prompted for this is like 8 years.